python/cudf_polars/cudf_polars/dsl/expressions/datetime.py

```diff
@@ -116,7 +116,10 @@ def __init__(
         self.name = name
         self.children = children
         self.is_pointwise = True
-        if self.name not in self._COMPONENT_MAP:
+        if (
+            self.name not in self._COMPONENT_MAP
+            and self.name is not TemporalFunction.Name.ToString
```

Comment:
Given your other PRs adding more datetime functionality, maybe it would be good to define a `_VALID_OPS` `set` on the class to union the `_COMPONENT_MAP` and supported `TemporalFunction.Name`s that we can check here